### DISCUSSION TITLE
v2.0.1リリースについて気づいたことがあれば

### DISCUSSION THREAD

**Discussion by keiji** [cocoa-mhlw/cocoa#1032] [General]
v2.0.1リリースは、v2.0.0で見つかった軽微な不具合の修正が含まれます。

 * #928 接触データのエクスポート機能を追加する
 * #980 接触一覧画面のタイムゾーンに関する表記を改善
 * #632 マルチユーザー（プライマリーユーザーでない）環境でクラッシュする
 * #987 [Android] 言語設定を変えてCOCOAを開くとApiExceptionが発生することがある
 * #967 「過去14日間の接触」画面を改善する
 * #966 「処理番号の取得方法」で説明している有効期限に誤りがある
 * #974 接触の可能性の通知は出るが「陽性者との接触は確認されませんでした」と表示される
 * #986 [A11y] 動作状況の読み上げの順番を修正する

v2.0.1リリースにあたって、コントリビューターの皆様から様々なご提案をいただきました。
ご協力に感謝いたします。

**Full Changelog**: https://github.com/cocoa-mhlw/cocoa/compare/v2.0.0...v2.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/cocoa-mhlw/cocoa/releases/tag/v2.0.1'>v2.0.1</a>.</em>

**Comment by polygon0323** [comment]
#923 の確認を行いました。
Q&Aの「接触の有無はどのように知ることができますか？」のページでテキストが切れていました。
他のページは問題なさそうです。
![Screenshot_20220617-233544](https://user-images.githubusercontent.com/4208047/174429555-b7374884-3fe2-43d2-9dd3-16284268975f.png)

**Reply by keiji** [reply]
ありがとうございます！　次バージョンではより確実に修正できるようにします！
（Q&Aの一覧画面でもアクセシビリティ上の課題があります）
